### ISSUE TITLE
why didnt you just fork my code from my repo instead of copying?

### ISSUE THREAD

**Issue by aqreed** [ggruszczynski/pyVLM#3]
look this 3D thing is a good idea, but usually the **ethical** way to work is to collaborate, not to just copy with no references. That way we can all improve and learn...

**Comment by aqreed** [comment]
i leave this here ...

https://softwareengineering.stackexchange.com/questions/22255/whats-the-right-way-to-fork-re-use-code-from-an-open-source-project/22261#22261

https://opensource.stackexchange.com/questions/5828/where-would-i-give-credit-to-an-mit-licensed-project-in-an-ios-app

**Comment by ggruszczynski** [comment]
Hi Aqreed,
Under the _MIT license_ it is **ethically** correct to use the code:
 -Commercially,
 -Modificate it,
 -Distribute it,
 -Sell it,
 -Use privatelly.

I introduced the 3D version, and I used it privately. This is why it was not a fork.
Later on, I decided to open the repo and make the code public. 
I appreciate that you found the 3D version interesting. 
Moreover, I can admit that it is better to have one good code than a couple of little, independent projects. If you like, I can make a pull request...

best regards,
Grzegorz
